src/ray/common/ray_config_def.h

```diff
@@ -318,6 +318,15 @@ RAY_CONFIG(uint32_t, task_retry_delay_ms, 0)
 /// Duration to wait between retrying to kill a task.
 RAY_CONFIG(uint32_t, cancellation_retry_ms, 2000)
 
+/// Whether to start a background thread to import Python dependencies eagerly.
+/// When set to false, Python dependencies will still be imported, only when
+/// they are needed.
+RAY_CONFIG(bool, start_python_importer_thread, true)
```

Comment:
Any concern about not using the new code by default?

Reply:
I don't have a significant concern right now. The additional latency from importing lazily shouldn't be a big issue. We can enable this by default in a separate PR.